python/katana/local/analytics/_k_core.pyx

```diff
@@ -106,7 +107,7 @@ cdef class KCorePlan(Plan):
         return KCorePlan.make(_KCorePlan.Asynchronous())
 
 
-def k_core(Graph pg, uint32_t k_core_number, str output_property_name, KCorePlan plan = KCorePlan()) -> int:
+def k_core(Graph pg, uint32_t k_core_number, str output_property_name, KCorePlan plan = KCorePlan(), TxnContext txn_ctx=None) -> int:
```

Comment:
NIT: put spaces around the `=` in default values.

Reply:
Hmm.... I didn't put spaces around the `=` because at most of other places, the default values of other parameters were without spaces around their `=`-s (written by others). Do you want me to change them all to be with spaces, or just make `TxnContext` to be consistent with other parameters?

Reply:
Right now I just make them consistent with their surrounding code.